packages/@vue/cli-plugin-typescript/generator/template/tsconfig.json

```diff
@@ -27,6 +27,9 @@
     },
     "lib": [
       "es2015",
+      <%_ if (options.useTsWithBabel) { _%>
+      "esnext",
```

Comment:
When we include `esnext`, `es2015` is not needed because `esnext` lib file includes `es2015`.